### ISSUE TITLE
The Glow parameter gradient is always 0

### ISSUE THREAD

**Issue by wulaoshi** [y0ast/Glow-PyTorch#16]
When I input the Bert sentence vector to the glow model for training, I found that the gradients of the parameters of the glow model are all 0, while the underlying Bert parameters have gradient values.

**Comment by y0ast** [comment]
This sounds like a bug with your implementation! I am sorry but this isn't the place for that, maybe try the pytorch forums?

**Comment by wulaoshi** [comment]
> This sounds like a bug with your implementation! I am sorry but this isn't the place for that, maybe try the pytorch forums?

Thank you, I have fixed the BUG.
